### PR TITLE
Fix missing licence.

### PR DESCRIPTION
**Pull request by pan-**

### CODE REVIEW

**Comment by pan-** [comment]
@0xc0170 could you review this ?

File: nordic-nrf51822-armcc/scripts/memory_info.py [hunk 1]
```diff
@@ -1,5 +1,19 @@
 #! /usr/bin/env python
 
+# Copyright (c) 2015-2016 ARM Limited
+#
```
Comment (rgrover) [inline nordic-nrf51822-armcc/scripts/memory_info.py:5]: add `SPDX-License-Identifier: Apache-2.0`

File: nordic-nrf51822-gcc/scripts/memory_info.py [hunk 2]
```diff
@@ -1,5 +1,19 @@
 #! /usr/bin/env python
 
+# Copyright (c) 2015-2016 ARM Limited
+#
```
Comment (rgrover) [inline nordic-nrf51822-gcc/scripts/memory_info.py:5]: same as above

**Comment by pan-** [comment]
I have added the `SPDX-License-Identifier: Apache-2.0` on both files. 
I merge.

**Pull request merged**